src/os/macos.rs

```diff
@@ -157,3 +165,27 @@ impl MonitorIdExt for MonitorId {
         self.inner.get_nsscreen().map(|s| s as *mut c_void)
     }
 }
+
+#[repr(i64)]
```

Comment:
Are these actually supposed to be `i64`, or are they an architecture-dependent size? I'd appreciate a comment including the `NSWhatever` name of the enum.

Reply:
It's an `i64` based on the current `NSVisualEffectView.setMaterial` method signature. The method takes a c-style enum, which is represented as an `i64`. I don't think this is set in stone but I doubt it will ever change.

I have added some valuable documentation in my new commit.